src/android/pspdfkit.gradle

```diff
@@ -18,4 +18,5 @@ repositories {
 
 dependencies {
     implementation "com.pspdfkit:$pspdfkitMavenModuleName:$pspdfkitVersion"
+    implementation "androidx.appcompat:appcompat:1.4.1"
```

Comment:
Was there any reason for adding `appcompat:1.4.1`?

Reply:
No. Just testing things out. Feel free to remove it.